packages/orbit-components/src/InputGroup/InputGroup.stories.tsx

```diff
@@ -111,32 +121,22 @@ export const PhoneNumber = ({
       />
       <InputField placeholder={placeholder} maxLength={11} value={inputValue} />
     </InputGroup>
-  );
-};
-
-PhoneNumber.story = {
-  name: "Phone number",
-
-  parameters: {
-    info: "Some description about this type of InputGroup in general.",
+  ),
+
+  args: {
+    flex: ["0 0 110px", "1 1 100%"],
+    selectValue: selectOptionsPhoneNumber[0].value,
+    customValueText: "+420",
+    placeholder: "e.g. 123 456 789",
+    inputValue: "",
   },
-};
 
-PhoneNumber.args = {
-  flex: ["0 0 110px", "1 1 100%"],
-  error: "",
-  help: "",
-  selectValue: selectOptionsPhoneNumber[0].value,
-  customValueText: "+420",
-  placeholder: "e.g. 123 456 789",
-  inputValue: "",
-};
-
-PhoneNumber.argTypes = {
-  selectValue: {
-    options: selectOptionsPhoneNumber.map(opt => opt.value),
-    control: {
-      type: "select",
+  argTypes: {
+    selectValue: {
+      options: selectOptionsPhoneNumber.map(opt => opt.value),
+      control: {
+        type: "select",
+      },
```

Comment:
Switching this value does not do anything to the UI. Shouldn't it change the number prefix?

<img width="406" alt="image" src="https://github.com/user-attachments/assets/c2d903a8-8f5a-4787-822c-728f4460f32e">

Reply:
It is changing the number prefix, but it' being overwritten by the `customValueText` value 🙈 As it didn't really make sense to have it there, I removed it. Also the CountryFlag code value was hardcoded, so I added it into `selectOptionsPhoneNumber`. Changing the value now should be changing the number prefix as well as the flag.